viper.go

```diff
@@ -148,14 +149,20 @@ func (b *Bundle) provideViper(ctx context.Context, flagSet *pflag.FlagSet) (_ *v
 
 	var configFile string
 	if configFile, err = flagSet.GetString("config"); err != nil {
-		return nil, err
+		return nil, fmt.Errorf("flag error: flag config not setup: %v", err)
 	}
 
 	if len(configFile) > 0 {
 		b.viper.SetConfigFile(configFile)
 	}
 
-	return b.viper, b.viper.ReadInConfig()
+	err = b.viper.ReadInConfig()
+	if err != nil {
+		return nil, fmt.Errorf("read config file error: %v. config file: '%s'",
```

Comment:
Better use error wrapping. Please change the fmt.Errorf arguments to `unable to read config file : %s : %w`,  configFile, err